### PR TITLE
Replace recent messages rollover with recent contacts using segment data from sprint

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: src/components/flow/exit/__snapshots__/Exit.test.tsx.snap [hunk 1]
```diff
@@ -145,46 +145,72 @@ exports[`ExitComp shows missing localization 1`] = `
...
           <div
-            class="message"
+            class="row"
           >
             <div
-              class="text"
+              class="contact"
             >
+              <a
+                href="undefinedeaea6433-d423-41dd-94a9-075de20f322d"
```
Comment (ericnewcomer) [inline src/components/flow/exit/__snapshots__/Exit.test.tsx.snap:171]: We should probably give our tests some placeholder endpoints so these look more right.

**Comment by ericnewcomer** [comment]
...additionally, on what branch do we have the associated endpoint configured? I'm trying to run this locally so I can see how it looks.

**Comment by rowanseymour** [comment]
@ericnewcomer https://github.com/nyaruka/rapidpro/pull/3638

**Comment by ericnewcomer** [comment]
@rowanseymour I got pretty confused with your merge which happened after my changes. Can you reapply on https://github.com/nyaruka/floweditor/pull/1030 ?

**Pull request closed without merging**